osm2gtfs/core/cache.py

```diff
@@ -49,7 +49,7 @@ def write_file(name, content):
         """
         if not os.path.isdir("data"):
             os.mkdir("data")
-        with open(os.path.join("data", name), "w") as f:
+        with open(os.path.join("data", name), "wb") as f:
```

Comment:
For my culture, why is it useful to use binary mode here ? :thinking:

Reply:
In `configuration.py` when the `schedule_source` value in configuration file points to a URL, the `urlopen()` method returns a bytes object. Then the schedule data is send to `Cache.write_file()`. Adding the binary mode in the `open()` makes  `write()` work with a bytes object as argument. If the case is the schedule file is already cached, actually the `Cache.read_file()` open it with binary mode. For consistency in `configuration.py` I add the binary mode for the case in which the file is local.

Reply:
ok, thanks for the explanation